src/pages/workspaces/workspace/analysis/modals/AnalysisModal.test.ts

```diff
@@ -19,6 +18,7 @@ import {
 import { AppTool, getToolLabelFromFileExtension, ToolLabel, tools } from 'src/pages/workspaces/workspace/analysis/tool-utils'
 import { asMockedFn } from 'src/testing/test-utils'
 
+import { reportError } from '../../../../../libs/error'
```

Comment:
curious... yarn lint cmd auto-fixed these imports for correct order, and did this relative pathing change here and in a few other files...